styleguide.config.js

```diff
@@ -33,6 +33,16 @@ module.exports = {
       components: './src/components/**/*.js',
       sectionDepth: 2,
     },
+    {
+      name: 'Xhr',
+      content: './src/xhr/content.md',
+      components: './src/xhr/components/**/*.js',
+    },
+    {
+      name: 'Nora',
```

Comment:
With this approach, we could potentially also add something like Marketing, perhaps for any off-brand components that are only meant for usage in landing pages and such.